dockerfiles/cli/scripts/openshift/deploy_che.sh

```diff
@@ -254,7 +254,10 @@ CHE_IMAGE_SANITIZED=$(echo "${CHE_IMAGE}" | sed 's/\//\\\//g')
 echo
 if [ "${OPENSHIFT_FLAVOR}" == "minishift" ]; then
   echo "[CHE] Deploying Che on minishift (image ${CHE_IMAGE})"
-  curl -sSL http://central.maven.org/maven2/io/fabric8/tenant/apps/che/"${OSIO_VERSION}"/che-"${OSIO_VERSION}"-openshift.yml | \
+  DEFAULT_CHE_DEPLOYMENT_URL=https://gist.githubusercontent.com/garagatyi/20e63a0d3c7b1a2a82a262eb2f8bcde1/raw/2716217b5dfa67c04283883b1df85fae364aadd4/che-spi-openshift.yml
```

Comment:
could we use another stuff than a gist ?
updating the gist will result in a new URL so maybe we need to store the file locally ?

Reply:
@l0rd WDYT if we put configuration of openshift objects into repo?

Reply:
@garagatyi +1

Reply:
done

Reply:
@garagatyi there are warnings on https://www.shellcheck.net/

Reply:
I fixed only one.
The one about `cat` is not so obvious that it would look better.
Others are false positives.